src/include/aerospike/as_exp.h

```diff
@@ -2967,11 +2969,13 @@ as_exp_destroy_base64(char* base64)
  * The results of the evaluation of the modifying expression will replace the
  * selected map and the changes written back to storage.
  *
- * @param Pointer to a CDT context.
- * @param Return type specifier (e.g., AS_EXP_TYPE_MAP).
- * @param Expression to apply.
- * @param Flags.
- * @param Bin to which the application applies.
+ * @param __ctx      Pointer to a CDT context.
+ * @param __rtype    Return type specifier (e.g., AS_EXP_TYPE_MAP).
+ * @param __mod_exp  Expression to apply.
+ * @param __flags    Flags (see enum as_cdt_modify_flags).
+ * @param __bin      Bin to which the application applies.
```

Comment:
```suggestion
 * @param __bin      Bin expression to which the application applies.
```

Forgot to mention -- I believe the bin parameter should be a bin expression; if I try to pass in a string, I think I get error code 4 from the server

If someone else can confirm, that would be great

Reply:
I believe you're right, because it also sets the type of the bin as well.  Thanks!